src/Controller/AttributesController.php

```diff
@@ -96,7 +94,8 @@ public function __invoke(Request $request): Response
             }
 
             $this->entityManager->flush();
-            $this->flashBag->add('success', $this->translator->trans('akeneo.ui.admin.changes_successfully_saved'));
+            Assert::isInstanceOf($request->getSession(), Session::class);
```

Comment:
un peu relou de faire cette assertion... Peut-être que ça pourrait être une erreur a ignorer dans phpstan ?

Reply:
Okay, i can do that ;)